### ISSUE TITLE
Remember the state of popped up menus

### ISSUE THREAD

**Issue by prati0100** [prati0100/git-gui#13]
Menus can be popped out by using the "line" at the top of every menu item under the top menu bar. It would be nice to remember where they were propped up when git-gui is closed, and their state restored when it is re-started.

Also consider adding an option to disable this behaviour in case some people don't like it.

**Comment by PhilipOakley** [comment]
> Menus can be popped out by using the "line" at the top of every menu item under the top menu bar.

I didn't even know that this was a feature / available. Something for the man page ;-)

**Comment by prati0100** [comment]
Yep, I just discovered this when messing around. AFAIK, this is provided by Tk, and we don't do anything special for it. It can be turned on/off via the `-tearoff` option.

Something for the man page indeed.

**Comment by PhilipOakley** [comment]
glad to see it wasn't just me..
